src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/PrivateDataRetrievalService.java

```diff
@@ -53,8 +56,13 @@ private ManagingOfficerListDataApi getManagingOfficerData(String companyNumber)
 
       ApiLogger.debug("Retrieving Managing Officer data for Company Number ", logMap);
```

Comment:
this debug is saying retrieving when it's already retrieved. Can we drop this debug please? Add log as for BOs if we need test evidence.